.github/workflows/pre-release-pr.yml

```diff
@@ -0,0 +1,83 @@
+name: Release test pre-release version
+
+on:
+  repository_dispatch:
+    types: [pre-release-command]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.client_payload.pull_request.head.sha }}
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
```

Comment:
Do we care about using a matrix for different python versions?

Reply:
What would be the point? 🤔 seeing if the library builds with different python versions? I think we already have the unit-tests for that

Reply:
we should use latest :D